kingpin/actors/base.py

```diff
@@ -381,8 +358,11 @@ def _check_condition(self):
         the value of self._condition is a string "False" or string "0".
         """
 
-        check = self.str2bool(self._condition)
-        self.log.debug("Condition %s evaluates to %s" % (self._condition, check))
+        try:
+            check = utils.str2bool(self._condition)
+            self.log.debug("Condition %s evaluates to %s" % (self._condition, check))
+        except ValueError as e:
+            raise exceptions.InvalidOptions(e.args) from e
```

Comment:
Added try-except to convert the raised error to maintain correct raise types outside function.